README.md

```diff
@@ -1,18 +1,89 @@
 # The Foreman Release Engineering scripts
 
-These are the scripts I use while releasing [The Foreman](https://theforeman.org). They automate most of the [release process](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb).
+A collection of small scripts, adhering to the [Unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy), used to release [The Foreman](https://theforeman.org). They automate parts of the [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb) processes. It is also used for the [Katello](https://theforeman.org/plugins/katello/) [branch](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/branch.md.erb) and [release](https://github.com/theforeman/tool_belt/blob/master/procedures/katello/release.md.erb).
+
+The most important environment variables are `PROJECT` and `VERSION`.
+
+## Procedures
+
+Procedures are kept in [tool_belt](https://github.com/theforeman/tool_belt) with small wrappers in this repository.
+
+### Roles
+
+Both the branch and release procedures defines two roles: an owner and an engineer. Someone can be both owner and engineer at the same time, and it is preferable for a fast process. Having both in (very) different timezones can lengthen the process. Sometimes it's just not possible and multiple people perform the procedures. Below is a listing of access used.
+
+#### Foreman release owner
+
+* Belong to the [Foreman GitHub release-managers team](https://github.com/orgs/theforeman/teams/release-managers)
```

Comment:
+ access to redmine to do the "new version", "close version" things there?
+ access to translations tool to fetch these?

Reply:
Good point on both.